### PR TITLE
various formulae: style fixes

### PR DESCRIPTION
**Pull request by nandahkrishna**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This PR is required for Homebrew/brew#11626. Some changes were made with `brew style --fix`, but others had to be done manually (where autocorrect wouldn't work properly).

Formulae where the changes weren't only style changes (but straightforward), I tried to build and test the formula locally. If we'd rather run CI here, I'll remove the `CI-syntax-only` label.

### CODE REVIEW

File: Formula/blaze.rb
Comment (carlocab) [inline Formula/blaze.rb:48]: This is not nice. (Not a blocker, just expressing unhappiness over this 😄)

**Comment by carlocab** [comment]
I'd rather run CI on this. Not all of these changes are obviously NFC.

**Comment by nandahkrishna** [comment]
Not urgent and the CI queue being backed up was my main concern. I'll mark this as "do not merge" until we get a chance to run CI.

File: Formula/cidrmerge.rb
Comment (bayandin) [inline Formula/cidrmerge.rb:2]:
```suggestion
  desc "CIDR merging with network exclusion"
```

File: Formula/blaze.rb
Comment (nandahkrishna) [reply Formula/blaze.rb:48]: Agreed, many of these were made with `brew style --fix` but I'll take a look in some time to modify the not-so-nice-looking ones.

File: Formula/fdroidserver.rb
Comment (carlocab) [inline Formula/fdroidserver.rb:356]: This is also not nice. Is it really too long to be one line?

File: Formula/fn.rb
Comment (carlocab) [inline Formula/fn.rb:33]: 👎

File: Formula/httpd.rb
Comment (carlocab) [inline Formula/httpd.rb:29]: Not sure this makes sense but ok.

File: Formula/ocrmypdf.rb
Comment (carlocab) [inline Formula/ocrmypdf.rb:141]: Terrible. Though I think we can fix this by doing `depends_on "pillow"` instead.

File: Formula/pmdmini.rb
Comment (carlocab) [inline Formula/pmdmini.rb:32]: The right fix here is to not pass one long string to `system`. We can use `*Dir["obj/*.o"]` to do the globbing.

File: Formula/speech-tools.rb
Comment (carlocab) [inline Formula/speech-tools.rb:74]: We should probably just extract the command we want to run into a string and pass that to `shell_output`.

File: Formula/sqlcipher.rb
Comment (carlocab) [inline Formula/sqlcipher.rb:29]: 👎

File: Formula/volatility.rb
Comment (carlocab) [inline Formula/volatility.rb:166]: Also awful, but we can `depends_on "pillow"` instead

File: Formula/weboob.rb
Comment (carlocab) [inline Formula/weboob.rb:120]: Same with the other `pillow` formulae

**Comment by carlocab** [comment]
Most of these are improvements but some of them are... not.

File: Formula/volatility.rb
Comment (bayandin) [reply Formula/volatility.rb:166]: `volatility` uses python2 🙁

File: Formula/ocrmypdf.rb
Comment (bayandin) [reply Formula/ocrmypdf.rb:141]: Will be done in https://github.com/Homebrew/homebrew-core/pull/80458

File: Formula/weboob.rb
Comment (bayandin) [reply Formula/weboob.rb:120]: Done in https://github.com/Homebrew/homebrew-core/pull/80307

File: Formula/fdroidserver.rb
Comment (bayandin) [reply Formula/fdroidserver.rb:356]: Will be fixed in https://github.com/Homebrew/homebrew-core/pull/80457

File: Formula/fn.rb
Comment (bayandin) [reply Formula/fn.rb:33]:
```ruby
response = {
  "id":"01CQNY9PADNG8G00GZJ000000A",
  "name":"myapp",
  "created_at":"2018-09-18T08:56:08.269Z",
  "updated_at":"2018-09-18T08:56:08.269Z",
}.to_json
```

File: Formula/volatility.rb
Comment (carlocab) [reply Formula/volatility.rb:166]: Sad times. Maybe we can also build `pillow` for Python2...

**Comment by nandahkrishna** [comment]
I'll rebase this once the Pillow-related PRs are merged. I'll also manually edit the ones that autocorrect spoilt, to the best of my ability 😅.

File: Formula/volatility.rb
Comment (bayandin) [reply Formula/volatility.rb:166]: > Maybe we can also build pillow for Python2...
Oh, let's don't do it 🙂 

Also `volatility` 3 should support `python3`, and it seems the first version of it has been releasere (at least tagget): https://github.com/volatilityfoundation/volatility3, not sure about its status though and if we can replace current version with the new one

File: Formula/httpd.rb
Comment (nandahkrishna) [reply Formula/httpd.rb:29]: It seems like a RuboCop bug, even with the latest version (1.18.2). I'll raise an issue with them.

**Comment by nandahkrishna** [comment]
ARM:
```
Error: 4 failed steps!
brew install --build-from-source flow
brew install --build-from-source hashlink
brew install --build-from-source hyperkit
brew install --build-from-source infer
```
None have ARM bottles, so this is expected.

**Comment by MikeMcQuaid** [comment]
> Formulae where the changes weren't only style changes (but straightforward), I tried to build and test the formula locally. If we'd rather run CI here, I'll remove the `CI-syntax-only` label.

I'm fine with using `CI-syntax-only` here, personally. I trust `rubocop` to do these autocorrects correctly.

**Comment by Moiss27** [comment]
HWhat is wrong with you people, I just told you I don't know what the hell any of you are talking about, I never signed up for any of this shit. What is the deal here,

________________________________

**Comment by nandahkrishna** [comment]
ARM Big Sur:
```
Error: 4 failed steps!
brew install --build-from-source flow     # no ARM bottle
brew install --build-from-source hashlink # no ARM bottle
brew install --build-from-source hyperkit # no ARM bottle
brew install --build-from-source infer    # no ARM bottle
```
✅ All expected.

Intel Big Sur:
```
Error: 4 failed steps!
brew install --build-from-source hyperkit        # Known to be broken, no Big Sur bottle
brew test --retry --verbose hyperkit             # Same as above
brew install --build-from-source infer           # Need to check
brew install --verbose --build-bottle volatility # Python2 has no venv module, changes not related to failure (Homebrew/brew#10039)
```
🟡 Need to check `infer`.

Catalina:
```
Error: 4 failed steps!
brew test --verbose fn                           # Connection failure, looks like a false positive
brew install --build-from-source hyperkit        # See Big Sur comment
brew test --retry --verbose hyperkit             # Same as above
brew install --verbose --build-bottle volatility # See Big Sur comment
```
✅ 1 false positive, others expected.

Mojave:
```
Error: 4 failed steps!
brew audit molten-vk --online --git --skip-style # Network-related stable URL not reachable (re-checked, passed)
brew install --build-from-source hyperkit        # See Big Sur comment
brew test --retry --verbose hyperkit             # Same as above
brew install --verbose --build-bottle volatility # See Big Sur comment
```
✅ 1 network issue, others expected.

File: Formula/ocaml.rb
Comment (nandahkrishna) [inline Formula/ocaml.rb:35]: I'm unsure why `infer` failed to build on Intel Big Sur. It was built/tested only because of `ocaml`, which has a syntax-only change in a comment.

If we can ignore this failure, the PR should be good to merge, I think.

**Review by carlocab — APPROVED** [review]

File: Formula/ocaml.rb
Comment (carlocab) [reply Formula/ocaml.rb:35]: Works for me. Thanks @nandahkrishna!

**Comment by nandahkrishna** [comment]
Rebased and merged. Thanks for the feedback everyone!

**Pull request merged**
